pages/common/git-rm.md

```diff
@@ -0,0 +1,15 @@
+# git rm
+
+> Remove files from the working tree and from the index
+
+- Remove file and add its deletion to next commit
```

Comment:
How about "Delete the file locally, and set it to be removed from the repository on the next commit"?